snippets/visualbasic/VS_Snippets_CLR_System/system.text.regularexpressions.regex.class/vb/regex_example1.vb

```diff
@@ -23,9 +23,12 @@
```

Comment:
NIT: You are correctly fixing the issue so that `?` now has the intended meaning (optional, instead of lazily), but you are also introducing a capturing group by doing so. We could opt either to document that in the description saying that the piece `(\d{2})?` is a `Capturing group 1: Match two decimal digits zero or one time.` or alternatively we can change this line to be `pattern += "(?:\d{"` which would turn this group into a non-capturing group which is only used to be able to wrap the `?`